doc/getstarted/build_and_install/build_from_source.md

```diff
@@ -15,13 +15,15 @@ cd paddle
 
 ## <span id="requirements">Requirements</span>
 
-To compile the source code, your computer must be equipped with GCC >=4.6 or Clang compiler.
+To compile the source code, your computer must be equipped with the following dependencies.
+
 ### Dependencies
 
+- **Compiler**: GCC >= 4.8 or Clang >= 3.3 (AppleClang >= 5.1)
```

Comment:
Yeah, Clang is supported in PaddlePaddle. Clang 3.4 is also supported by the latest CUDA Toolkits.